graphql/resolvers.go

```diff
@@ -712,6 +712,14 @@ func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string)
 	return result, nil
 }
 
+func (r *queryResolver) CommitQueue(ctx context.Context, id string) (*restModel.APICommitQueue, error) {
+	commitQueue, err := r.sc.FindCommitQueueByID(id)
+	if err != nil || commitQueue == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Error finding commit Queue for `%s`: %s", id, err))
```

Comment:
we should probably return the equivalent of 500 when err != nil, and 404 when commitQueue == nil

Reply:
also I thinkthat you should err.Error() to get a string, rather than %s, which is weird from a typing perspective.

Reply:
If no commitQueue is found there will always be an error. since `FincCommitQueueByID` will always return an error if  it cant find a commit queue. https://github.com/evergreen-ci/evergreen/blob/28c66fda6b8f4013ed4e0b3548444b5cf0d09245/rest/data/commit_queue.go#L78

Reply:
Im not sure if there is a way to return two separate error messages with out actually looking at the error message returned from  `FindCommitQueueByID` and returning a different Error code based on the error message itself. so
```
if contains(err, "no commit queue found"){
//Return 404
}else{
// Return 500
}
```
What do you think?

Reply:
a lot of our APIs return nil errors when things aren't found (and just return nil for the other return value), so I thought this API is like that. If you check that those dimensions are collapsed in the API that you're consuming we can't really distinguish it here.